src/main/java/games/strategy/engine/message/RemoteMethodCallResults.java

```diff
@@ -49,8 +49,8 @@ public void writeExternal(final ObjectOutput out) throws IOException {
 
   @Override
   public void readExternal(final ObjectInput in) throws IOException, ClassNotFoundException {
-    final boolean rVal = in.read() == 1;
-    if (rVal) {
+    final boolean hasReturnValue = in.read() == 1;
+    if (hasReturnValue) {
```

Comment:
The variable is not needed here, the boolean expression can be moved inside the if statement

Reply:
This point should actually be adressed within the scope of this PR.